pkg/services/reputation/local/route/managers/deps.go

```diff
@@ -0,0 +1,14 @@
+package managers
+
+import (
+	"github.com/nspcc-dev/neofs-api-go/pkg/netmap"
+	"github.com/nspcc-dev/neofs-api-go/pkg/reputation"
+)
+
+// ManagerBuilder defines an interface for providing a list
+// of Managers for specific epoch and node.
+type ManagerBuilder interface {
+	// BuildManagers must compose list of managers,
+	// using network map of particular epoch.
```

Comment:
Lets remove `using network map` since this is an implementation detail, not a requirement for all implementations.

Reply:
removed